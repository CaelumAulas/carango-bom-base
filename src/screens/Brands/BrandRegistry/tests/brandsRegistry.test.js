import React from 'react';
import { render } from '@testing-library/react';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';

import BrandsRegistry from '..';

describe('BrandsRegistry component', () => {
  it('should match the snapshot', () => {
    const history = createMemoryHistory();

    const { container } = render(
      <Router history={history}>
        <BrandsRegistry />
      </Router>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
