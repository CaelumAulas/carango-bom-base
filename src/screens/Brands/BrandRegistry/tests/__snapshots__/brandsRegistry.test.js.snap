// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrandsRegistry component should match the snapshot 1`] = `
<form>
  <div
    class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
  >
    <label
      class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-error Mui-error Mui-required Mui-required"
      data-shrink="false"
      for="brand"
      id="brand-label"
    >
      Marca
      <span
        aria-hidden="true"
        class="MuiFormLabel-asterisk MuiInputLabel-asterisk Mui-error Mui-error"
      >
         
        *
      </span>
    </label>
    <div
      class="MuiInputBase-root MuiOutlinedInput-root Mui-error Mui-error MuiInputBase-fullWidth MuiInputBase-formControl"
    >
      <input
        aria-invalid="true"
        class="MuiInputBase-input MuiOutlinedInput-input"
        id="brand"
        name="brand"
        required=""
        type="text"
        value=""
      />
      <fieldset
        aria-hidden="true"
        class="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
      >
        <legend
          class="PrivateNotchedOutline-legendLabelled-3"
        >
          <span>
            Marca
             *
          </span>
        </legend>
      </fieldset>
    </div>
    
  </div>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary Mui-disabled Mui-disabled"
    disabled=""
    tabindex="-1"
    type="submit"
  >
    <span
      class="MuiButton-label"
    >
      Cadastrar
    </span>
  </button>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiButton-label"
    >
      Cancelar
    </span>
    <span
      class="MuiTouchRipple-root"
    />
  </button>
</form>
`;
